examples/gh-org-runner-mig-native-simple/outputs.tf

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "mig_instance_group" {
+  description = "The instance group url of the created MIG"
+  value       = module.runner_mig.mig_instance_group
+}
+
+output "mig_name" {
+  description = "The name of the MIG"
+  value       = module.runner_mig.mig_name
+}
+
+output "service_account" {
+  description = "Service account email for GCE"
+  value       = module.runner_mig.service_account
+}
+
+output "mig_instance_template" {
+  description = "The name of the MIG Instance Template"
+  value       = module.runner_mig.mig_instance_template
+}
+
+output "gh_secret_id" {
+  description = "Secret Manager ID of the github secrets (token, repo_name,repo_owner)"
+  value       = module.runner_mig.gh_secret_id
+}
```

Comment:
@raj-saxena small nit, we can remove this. This is something the user supplies plus is  secret so we dont need to output this.

Reply:
Updated https://github.com/terraform-google-modules/terraform-google-github-actions-runners/pull/37/commits/4fff91f5fc3680ff60fb5fceaddc08fc24d16e1d